.github/workflows/canary-integration-test.yml

```diff
@@ -1528,8 +1528,75 @@ jobs:
       - name: set Ceph version in CephCluster manifest
         run: tests/scripts/github-action-helper.sh replace_ceph_image  "deploy/examples/cluster-test.yaml" "${{ github.event.inputs.ceph-image }}"
 
-      - name: run RGW multisite test
-        uses: ./.github/workflows/rgw-multisite-test
+      - name: setup cluster resources
```

Comment:
Did anything change in this script, or just moved from the other file?

Reply:
There were some changes.  The inlinification is in separate commit to try to make it easy to view the changes to the workflow. https://github.com/rook/rook/pull/14842/commits/92d9f994c2bdced5b4bebed8790be8c1c49b53c0